src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -211,14 +221,20 @@ private void TabView_TabDragCompleted(TabView sender, TabViewTabDragCompletedEve
 
 		private async void TabView_TabDroppedOutside(TabView sender, TabViewTabDroppedOutsideEventArgs args)
 		{
-			if (sender.TabItems.Count == 1)
-			{
+			InteropHelpers.GetCursorPos(out var droppedPoint);
+			var droppedTime = DateTimeOffset.UtcNow;
+			var dragTime = droppedTime - dragStartTime;
+			var dragDistance = Math.Sqrt(Math.Pow((dragStartPoint.X - droppedPoint.X), 2) + Math.Pow((dragStartPoint.Y - droppedPoint.Y), 2));
+
+			if (sender.TabItems.Count == 1 ||
+				dragTime.TotalSeconds < 1 &&
+				dragDistance < 100)
```

Comment:
The second condition (after `||`) should be wrapped with a parenthesis or replace `&&` with `||`

Reply:
I pushed a commit with this change